.github/workflows/smoke-test.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        k8s: [ v1.23.13, v1.27.1 ]
+        k8s: [ v1.23.13, v1.24.15, v1.25.11, v1.26.6, v1.27.1 ]
```

Comment:
keep just 3 versions here: v1.23.13, v1.25.11 and v1.27.1

Reply:
changed